dipy/sims/voxel.py

```diff
@@ -288,7 +299,265 @@ def multi_tensor(gtab, mevals, S0=100, angles=[(0, 0), (90, 0)],
+    >>> from dipy.data import get_data
+    >>> from dipy.core.gradients import gradient_table
+    >>> from dipy.io.gradients import read_bvals_bvecs
+    >>> fimg, fbvals, fbvecs = get_data('small_101D')
+    >>> bvals, bvecs = read_bvals_bvecs(fbvals, fbvecs)
+    >>> gtab = gradient_table(bvals, bvecs)
+    >>> mevals=np.array(([0.0015, 0.0003, 0.0003],[0.0015, 0.0003, 0.0003]))
+    >>> e0 = np.array([1, 0, 0.])
+    >>> e1 = np.array([0., 1, 0])
+    >>> dt, kt, S =  multi_tensor_dki(gtab, mevals)
+    
+    References
+    ----------
+    .. [1] R. Neto Henriques et al., "Exploring the 3D geometriy of the 
+           diffusion kurtosis tensor - Impact on the development of robust 
+           tractography procedures and novel biomarkers", NeuroImage (2015) 
+           111, 85-99.
+    """
+                             
+    if np.round(np.sum(fractions), 2) != 100.0:
+        raise ValueError('Fractions should sum to 100')
+
+    fractions = [f / 100. for f in fractions]
+
+    S = np.zeros(len(gtab.bvals))
+    
+    angles = np.array(angles)
+    if angles.shape[-1] == 3:
+        sticks = angles
+    else:
+        sticks = [sphere2cart(1, np.deg2rad(pair[0]), np.deg2rad(pair[1]))
+                  for pair in angles]
+        sticks = np.array(sticks)
+    
+    # computing a 3D matrice containing the individual DT components 
+    mD=np.zeros((len(fractions),3,3))
+    for i in range(len(fractions)):
+        R = all_tensor_evecs(sticks[i])
+        mD[i] = dot(dot(R, np.diag(mevals[i])), R.T)
+        
+    # compute DT
+    D=np.zeros((3,3))
+    for i in range(len(fractions)):
+        D=D+fractions[i]*mD[i]    
+    dt=np.array([D[0][0],D[1][1],D[2][2],D[0][1],D[0][2],D[1][2]])
+    
+    # compute KT
+    kt=np.zeros((15))
+    kt[0] = compute_Wijkl(mD, fractions, 0,0,0,0, D)
+    kt[1] = compute_Wijkl(mD, fractions, 1,1,1,1, D)
+    kt[2] = compute_Wijkl(mD, fractions, 2,2,2,2, D)
+    kt[3] = compute_Wijkl(mD, fractions, 0,0,0,1, D)
+    kt[4] = compute_Wijkl(mD, fractions, 0,0,0,2, D)
+    kt[5] = compute_Wijkl(mD, fractions, 0,1,1,1, D)
+    kt[6] = compute_Wijkl(mD, fractions, 1,1,1,2, D)
+    kt[7] = compute_Wijkl(mD, fractions, 0,2,2,2, D)
+    kt[8] = compute_Wijkl(mD, fractions, 1,2,2,2, D)
+    kt[9] = compute_Wijkl(mD, fractions, 0,0,1,1, D)
+    kt[10] = compute_Wijkl(mD, fractions, 0,0,2,2, D)
+    kt[11] = compute_Wijkl(mD, fractions, 1,1,2,2, D)
+    kt[12] = compute_Wijkl(mD, fractions, 0,0,1,2, D)
+    kt[13] = compute_Wijkl(mD, fractions, 0,1,1,2, D)
+    kt[14]=  compute_Wijkl(mD, fractions, 0,1,2,2, D)
+    
+    # compute S based on the DT and KT
+    S=single_diffkurt_tensors(gtab, S0, dt, kt, snr)
+
+    return dt, kt, S
+    
+def compute_Wijkl(Dc,frac,i,j,k,el,DT=None):
+    r""" # function that computes the elements of KT by especifying their 
+    four indexes i, j, k and el. Each index can be equal to x, y or z which are 
+    codified respectively by 0, 1 and 2. Simulations are based on 
```

Comment:
Unfortunately, defining a principal orientation of KT is not trivial (and what will this principal orientation mean on KT? - don't forget that KT is not an ellipsoid). KT have 15 elements: Wxxxx Wyyyy Wzzzz Wxxxy Wxxxz Wxyyy Wyyyz Wxzzz Wyzzz Wxxyy Wxxzz Wyyzz Wxxyz Wxyyz Wxyzz - what I wanted to say for codifying each element I use 0 for x, 1 for y and 2 for z.